packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -18,20 +20,61 @@ import 'package:flutter/widgets.dart';
 
 import 'colors.dart';
 import 'theme.dart';
-import 'thumb_painter.dart';
 
-// Examples can assume:
-// bool _lights = false;
-// void setState(VoidCallback fn) { }
+// Hand coded defaults eyeballed from iOS simulator on Mac.
```

Comment:
nit: include what version of iOS you are looking at.

Reply:
These were the previously-used defaults, and they didn't have a version. I think it's up to date though fidelity-wise, at least for those defaults, so I'll use the latest version I have.